model/service.go

```diff
@@ -54,6 +54,9 @@ type Service struct {
 	// service DNS name.  External services are name-based solution to represent
 	// external service instances as a service inside the cluster.
 	ExternalName string `json:"external"`
+
+	// ServiceAccounts specifies the service accounts that run the service.
+	ServiceAccounts []string `json:"serviceaccounts,omitempty"`
```

Comment:
There is an interface currently that provides `GetServiceAccounts(hostname)` for a service hostname. It's essentially aggregating accounts of instances. You could also add an optional property to a service instance that holds the account information, and then aggregate it yourself.